sql_generators/active_users_aggregates/__init__.py

```diff
@@ -0,0 +1,101 @@
+"""Generate active users aggregates per app."""
+import os
+from enum import Enum
+from pathlib import Path
+
+import click
+from jinja2 import Environment, FileSystemLoader
+
+from bigquery_etl.cli.utils import use_cloud_function_option
+from bigquery_etl.format_sql.formatter import reformat
+from bigquery_etl.util.common import render, write_sql
+
+THIS_PATH = Path(os.path.dirname(__file__))
+TABLE_NAME = "active_users_aggregates"
+
+
+class Browsers(Enum):
+    """Enumeration with browser names and equivalent dataset names."""
+
+    firefox_desktop = "Firefox Desktop"
+    fenix = "Fenix"
+    focus_ios = "Focus iOS"
+    focus_android = "Focus Android"
+    firefox_ios = "Firefox iOS"
+    klar_ios = "Klar iOS"
+
+
+@click.command()
+@click.option(
+    "--output-dir",
+    "--output_dir",
+    help="Output directory generated SQL is written to",
+    type=click.Path(file_okay=False),
+    default="sql",
+)
+@click.option(
+    "--target-project",
+    "--target_project",
+    help="Google Cloud project ID",
+    default="moz-fx-data-shared-prod",
+)
+@use_cloud_function_option
+def generate(target_project, output_dir, use_cloud_function):
+    """Generate per-app queries, views and metadata for active users and search counts aggregates.
```

Comment:
Would these tables have a schema file?

Reply:
Yes, the schema is automatically derived from the query and deployed the next day in DAG [bqetl_artifact_deployment](https://workflow.telemetry.mozilla.org/dags/bqetl_artifact_deployment/grid).